### PR TITLE
Add data_ref for data_column parameter

### PR DESCRIPTION
**Pull request by bernt-matthias**

### CODE REVIEW

**Comment by bernt-matthias** [comment]
@rmassei can you have a look at the problem here?

**Comment by rmassei** [comment]
Trying to solving it [here](https://github.com/Helmholtz-UFZ/galaxy-tools/pull/56), but without success. I will run some local test but it seems a problem with the R script and the input format.

File: tools/tox_tools/dose_responses/dose_response.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <tool id="dr_curve" name="Dose Response Curve for Toxicological Risk Assessment" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="23.0">
-    <description>Toxicity prediction tool - Dose response Curve</description>
+    <description>wit drc</description>
```
Comment (bernt-matthias) [inline tools/tox_tools/dose_responses/dose_response.xml:2]: wit -> with?

**Comment by rmassei** [comment]
Great testing fix
:v: ready to go  :v:

**Review by rmassei — APPROVED** [review]

**Pull request merged**